content/tests/components/recipes/details/RecipeDetails.test.js

```diff
@@ -35,6 +36,26 @@ describe("The `RecipeDetailForm` component", () => {
     jest
       .spyOn(NormandyAPI.prototype, "fetchRecipe")
       .mockImplementation(() => Promise.resolve(recipe));
+
+    jest
+      .spyOn(ExperimenterAPI.prototype, "fetchExperiment")
+      .mockImplementation(() =>
+        Promise.resolve({
+          public_description: "Try this new thing",
```

Comment:
It may be worthwhile to make this a factory, in `content/tests/factories/experimenter.ts`.